packages/core/content-type-builder/admin/src/components/FormModal/forms/index.js

```diff
@@ -121,12 +121,13 @@ const forms = {
           })
         : pluralNames;
 
-      const contentTypeShape = createContentTypeSchema(
-        takenNames,
-        reservedNames.models,
-        takenSingularNames,
-        takenPluralNames
-      );
+      const contentTypeShape = createContentTypeSchema({
+        usedContentTypeNames: takenNames,
+        reservedModels: reservedNames.models,
+        reservedCollectionNames: reservedNames.collectionNames,
```

Comment:
Just curious, from where do you get this reservedNames list?

Reply:
They come from `packages/core/content-type-builder/server/services/builder.js`

Reply:
Aham! I am trying to find the request to get that data (trying to learn a little bit of the frontend side)
could you point me where it is made?

Reply:
It seems to be managed by the `DataManagerProvider` [here](https://github.com/strapi/strapi/blob/cf49ddbbfc33fa6a0145f9ddf50677e31c1d711e/packages/core/content-type-builder/admin/src/components/DataManagerProvider/index.js#L97)